### DISCUSSION TITLE
Multi-threading, cmake/make compilation "make -jN"

### DISCUSSION THREAD

**Discussion by Sid8899** [microsoft/WSL#7043] [Q&A]
Hi !

I'm using wsl2, it support multi-threading ? for softwares ? I need to compile from source some toolkits with cmake/make, I think you are familiar with those !? The command "make -jN"  works or we should not use it in wsl2 ? N correspond to the number of threads

Regards

**Comment by therealkenc** [comment]
> The command "make -jN" works 

Yes.

**Reply by Sid8899** [reply]
Great ! and should we use "nproc"  in bash to get N ?
